crates/cxx-qt-lib/src/core/qvector/qvector_quuid.rs

```diff
@@ -0,0 +1,92 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
```

Comment:
wrong contributor, it should be you yes?

Reply:
Autogenerated from https://github.com/cxx-qt-lib/cxx-qt/blob/d79dd4c39db6fbd5e29cd38caed095c2a7022f91/crates/cxx-qt-lib/src/core/qvector/generate.sh#L16

Reply:
It might be a good idea to add a .gitattributes to [mark these files as autogenerated](https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github).

Reply:
Ah then I'll drop the comments for now. Mind opening an issue about looking into .gitattributes?

Reply:
Sure thing! #1150